next.config.js

```diff
@@ -1,5 +1,9 @@
 /** @type {import('next').NextConfig} */
+
+const isProd = process.env.NODE_ENV === 'production'
+
 const nextConfig = {
+  basePath: isProd ? '/openbc-web' : '',
```

Comment:
I noticed we did this at the bottom as well.

Instead of checking environment then setting basePath. Maybe we can move basePath into environment variables?

Reply:
No, because when in Development env, NextJs's base Path would be "".